File: MasterErp.Angular/src/app/components/GeneralAccounts/models/GeneralAccounts/ReceiptModel.ts
export interface ReceiptModel {
    receiptId: number | null;
    orderNumber: number | null;
    receiptNumber: number | null;
    paymentTypeId: number;
    receiptLedgerId: number | null;
    receiptTypeId: number | null;
    paymentOrderId: number | null;
    releaseDate: string;
    contactName: string,
    fromAccountId: number | null;
    bankAccountId: number | null;
    agencyTypeId: number;
    accountId: number | null;
    supplierId: number | null;
    customerId: number | null;
    employeeId: number | null;
    moneyAmount: number;
    docNumber: string;
    currencyId: number | null;
    description: string;
    workflowStatusId: number | null;
    workflowStatusNameAR: string;
    workflowStatusNameEN: string;
    createdBy: string;
    createdDate: string;
    modifiedBy: string;
    modifiedDate: string | null;
}
export interface ReceiptLedger {
    receiptLedgerId: number;
    startReceiptNumber: number;
    periodId: number;
    receiptLedgerTypeId: number;
    operationTypeId: number;
    isActive: boolean;
    isLocked: boolean;
    code: string;
    nameAR: string;
    nameEN: string;
    notes: string;
    insertUser: string;
    updateUser: string;
    insertDate: string;
    updateDate: string | null;
}

File: RetailStation.Entities/Common/Finance/GeneralAccounts/ReceiptModel.cs
using MasterErp.Entities.Models;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MasterErp.Entities.Common.Finance.GeneralAccounts
{
    public class ReceiptModel : CreatorModel
    {
        public int? ReceiptId { get; set; }
        public int? ReceiptNumber { get; set; }
        public int? OrderNumber { get; set; }
        public int? ReceiptLedgerId { get; set; }
        public int PaymentTypeId { get; set; }
        public int? JournalEntryId { get; set; }
        public int? CurrencyId { get; set; }
        public int? ReceiptTypeId { get; set; }
        public int? BankAccountId { get; set; }
        public int? CustomerId { get; set; }
        public int? EmployeeId { get; set; }
        public int? AgencyTypeId { get; set; }
        public int? FromAccountId { get; set; }
        public int? AccountId { get; set; }
        public int? SupplierId { get; set; }
        public int? PaymentOrderId { get; set; }
        public string PaymentTypeName { get; set; }
        public string ReceiptTypeName { get; set; }
        public string ReceiptLedgerName { get; set; }
        public DateTime ReleaseDate { get; set; }
        public string ContactName { get; set; }
        public string BankAccount { get; set; }
        public string ChequeNumber { get; set; }
        public string AgencyTypeName { get; set; }
        public string AccountName { get; set; }
        public string AccountNumber { get; set; }
        public string SupplierName { get; set; }
        public string CustomerName { get; set; }
        public string EmployeeName { get; set; }
        public double MoneyAmount { get; set; }
        public string DocNumber { get; set; }
        public string Currency { get; set; }
        public string Description { get; set; }
        public string WorkflowStatusNameEN { get; set; }
        public string WorkflowStatusNameAR { get; set; }
        public int? WorkflowStatusId { get; set; }
        public int? EntryId { get; set; }
        public int? EmployeeAdvanceId { get; set; }
        public int? EmployeeDueId { get; set; }
        public int? TotalCount { get; set; }

        public int? NextId { get; set; }
        public int? PreviousId { get; set; }
    }
}
